Codegen/src/Generators/DefaultGenerator.cs

```diff
@@ -895,15 +903,19 @@ private static void AdvanceToNextStateAndAdjustTokenProperties(TokensLine tempTo
                     while ((t = scanner.GetNextToken()) != null)
                     {
                         if (t.TokenType == TokenType.GLOBAL_STORAGE)
-                        {//Skip GLOBAL-STORAGE SECTION. tokens
+                        {//Skip and remove GLOBAL-STORAGE SECTION. tokens by generating spaces instead.
+                            sw.Write(new System.String(' ', t.Text.Length));
```

Comment:
Why do we need to generate space instead ?

Reply:
Because doing that fix the issue, it keeps every things at its original location.